examples/files.json

```diff
@@ -1,434 +1,424 @@
-		"webgl_simple_gi",
-		"webgl_tiled_forward",
-		"webgl_worker_offscreencanvas"
-	],
-	"webgl2": [
-		"webgl2_buffergeometry_attributes_integer",
-		"webgl2_buffergeometry_attributes_none",
-		"webgl2_materials_texture2darray",
-		"webgl2_materials_texture3d",
-		"webgl2_materials_texture3d_partialupdate",
-		"webgl2_multiple_rendertargets",
-		"webgl2_multisampled_renderbuffers",
-		"webgl2_rendertarget_texture2darray",
-		"webgl2_texture2darray_compressed",
-		"webgl2_ubo",
-		"webgl2_volume_cloud",
-		"webgl2_volume_instancing",
-		"webgl2_volume_perlin"
-	],
-	"webgpu": [
-		"webgpu_audio_processing",
-		"webgpu_compute",
-		"webgpu_cubemap_adjustments",
-		"webgpu_cubemap_mix",
-		"webgpu_depth_texture",
-		"webgpu_equirectangular",
-		"webgpu_instance_mesh",
-		"webgpu_instance_uniform",
-		"webgpu_lights_custom",
-		"webgpu_lights_selective",
-		"webgpu_loader_gltf",
-		"webgpu_materials",
-		"webgpu_nodes_playground",
-		"webgpu_particles",
-		"webgpu_rtt",
-		"webgpu_sandbox",
-		"webgpu_skinning",
-		"webgpu_skinning_instancing",
-		"webgpu_skinning_points",
-		"webgpu_sprites"
-	],
-	"webaudio": [
-		"webaudio_orientation",
-		"webaudio_sandbox",
-		"webaudio_timing",
-		"webaudio_visualizer"
-	],
-	"webxr": [
-		"webxr_ar_cones",
-		"webxr_ar_dragging",
-		"webxr_ar_hittest",
-		"webxr_ar_lighting",
-		"webxr_ar_paint",
-		"webxr_ar_plane_detection",
-		"webxr_vr_ballshooter",
-		"webxr_vr_cubes",
-		"webxr_vr_dragging",
-		"webxr_vr_handinput",
-		"webxr_vr_handinput_cubes",
-		"webxr_vr_handinput_profiles",
-		"webxr_vr_handinput_pointerclick",
-		"webxr_vr_handinput_pointerdrag",
-		"webxr_vr_handinput_pressbutton",
-		"webxr_vr_haptics",
-		"webxr_vr_layers",
-		"webxr_vr_panorama",
-		"webxr_vr_panorama_depth",
-		"webxr_vr_paint",
-		"webxr_vr_rollercoaster",
-		"webxr_vr_sandbox",
-		"webxr_vr_sculpt",
-		"webxr_vr_teleport",
-		"webxr_vr_video"
-	],
-	"games": [
-		"games_fps"
-	],
-	"physics": [
-		"physics_ammo_break",
-		"physics_ammo_cloth",
-		"physics_ammo_instancing",
-		"physics_ammo_rope",
-		"physics_ammo_terrain",
-		"physics_ammo_volume",
-		"physics_oimo_instancing"
-	],
-	"misc": [
-		"misc_animation_groups",
-		"misc_animation_keys",
-		"misc_boxselection",
-		"misc_controls_arcball",
-		"misc_controls_drag",
-		"misc_controls_fly",
-		"misc_controls_map",
-		"misc_controls_orbit",
-		"misc_controls_pointerlock",
-		"misc_controls_trackball",
-		"misc_controls_transform",
-		"misc_exporter_collada",
-		"misc_exporter_draco",
-		"misc_exporter_gltf",
-		"misc_exporter_obj",
-		"misc_exporter_ply",
-		"misc_exporter_stl",
-		"misc_exporter_usdz",
-		"misc_lookat"
-	],
-	"css2d": [
-		"css2d_label"
-	],
-	"css3d": [
-		"css3d_molecules",
-		"css3d_orthographic",
-		"css3d_periodictable",
-		"css3d_sandbox",
-		"css3d_sprites",
-		"css3d_youtube"
-	],
-	"svg": [
-		"svg_lines",
-		"svg_sandbox"
-	],
-	"tests": [
-		"webgl_furnace_test",
-		"webgl_pmrem_test",
-		"misc_uv_tests"
-	]
```

Comment:
Please undo the formatting changes to this file. There should be only a single addition in `files.json`.

Reply:
Whoops, sorry, prettier default formatting 😅 
pushed a new fix